app/Http/Controllers/ProductController.php

```diff
@@ -31,13 +36,15 @@ public function index(ProductRequest $request)
             });        
             
             if($request->input('paginate') == 'true'){
```

Comment:
Move the pagination logic to a separate method to keep the index method clean

Reply:
you mean that i should create a function in the product model consisting of the if-block's statements?